modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,28 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
+== 6.6.1 - 2023-08-02
+
+=== Added
+* New filtering option, `pad_empty_with_br`. Set to `true` to pad empty block elements with `<br>` tags instead of the `+&nbsp;+` character entity.
```

Comment:
```suggestion
* Restored filtering option, `pad_empty_with_br`. Set to `true` to pad empty block elements with `<br>` tags instead of the `+&nbsp;+` character entity.
```